From 7787086ef2dc0988df8855721052ab2e801119bf Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 27 Nov 2024 23:28:40 +0800
Subject: [PATCH] feat(util/linux.py): add AOSC OS optenv32 path (/opt/32) to
 multiarch_lib_folders

AOSC OS is a non-multiarch distribution which ships its 32-bit x86 runtime
(under the name optenv32) in /opt/32, and the "native" 64-bit x86-64
runtime in /usr/lib.

Add this path combo to make Lutris work on AOSC OS.
---
 lutris/util/linux.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lutris/util/linux.py b/lutris/util/linux.py
index ac53d3e6..44490b47 100644
--- a/lutris/util/linux.py
+++ b/lutris/util/linux.py
@@ -95,6 +95,7 @@ class LinuxSystem:  # pylint: disable=too-many-public-methods
         ("/usr/lib32", "/usr/lib64"),
         ("/lib/i386-linux-gnu", "/lib/x86_64-linux-gnu"),
         ("/usr/lib/i386-linux-gnu", "/usr/lib/x86_64-linux-gnu"),
+        ("/usr/lib", "/opt/32/lib"),
     ]
 
     soundfont_folders = [
-- 
2.47.1

